### PR TITLE
feat(website/calloutvue): display vuejs link in code tab

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: packages/gatsby-theme-styleguide/src/gatsby-components/CalloutVue/CalloutVue.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+const VueIcon = styled.svg`
+  margin-left: 0.5rem;
+  margin-right: 0.25rem;
+`
+
+const CalloutVue = ({ link }) => {
+  return (
+    <VueJSBlock>
+      <b>Available on: </b>
```
Comment (nherwegh) [inline packages/gatsby-theme-styleguide/src/gatsby-components/CalloutVue/CalloutVue.js:29]: can we make this link clickable?

**Review by nherwegh — CHANGES_REQUESTED** [review]
vueJS_spacing

here are some spacing indications

File: packages/gatsby-theme-styleguide/src/gatsby-components/CalloutVue/CalloutVue.js (hunk 1 above)
Comment (mohamedMok) [reply packages/gatsby-theme-styleguide/src/gatsby-components/CalloutVue/CalloutVue.js:29]: @nherwegh 
Sure we can, what should be clickable? 
The full block or the full text? 
rn only the display-external-link icon is clickable

Comment (nherwegh) [reply packages/gatsby-theme-styleguide/src/gatsby-components/CalloutVue/CalloutVue.js:29]: the great thing would be to make: 
"VueJS" & the icon clickable

**Review by tiloyi — APPROVED** [review]

**Review by nherwegh — APPROVED** [review]

**Pull request merged**